### ISSUE TITLE
After enrolling phrase is identified not the speaker.

### ISSUE THREAD

**Issue by prateek03jan** [Microsoft/Cognitive-SpeakerRecognition-Windows#5]
Thanks for your help.Now we are able to enroll a phrase by repeating it thrice.
It is possible to recognize a particular speaker?

**Comment by prateek03jan** [comment]
Can we add different phrases of our own customization apart from the provided list of phrases?

**Comment by momohs** [comment]
@prateek03jan At this moment, if you don't want to use the provided list of phrases, you can use the identification service instead of the verification service.

**Comment by MagSag** [comment]
What it is the difference between the identification service with the verification server? I am developping an application using xamarin and would like to know who to identify the voice of the person who is speaking; even if many people say the same word for instance the word "John" at the same time, the app should check if among these people (the voices) there is one that is known(should have access to a part of my app.

**Comment by prateek03jan** [comment]
I had the similar scenario occurring in my application. I mapped a person
entity with voice and image by keeping the mapping with unique id at my
server end. But the authenticity was valid for the phrase recognition but
it was easily possible to bypass the voice recognition in case you try with
changing your voice. So I kept the authenticity checking on 3 steps.
1. Face recognition for identifying the person.
2. Voice phrase out of the listed phrases.
3. A passkey validation at my server end as the confidence of the voice
recognition results were more than 80% in majority cases.
If you limit the threshold for the voice api recognition, say more than 80%
it will become too strict.

On 20-Mar-2017 9:20 am, "MagSag" <notifications@github.com> wrote:

What it is the difference between the identification service with the
verification server? I am developping an application using xamarin and
would like to know who to identify the voice of the person who is speaking;
even if many people say the same word for instance the word "John" at the
same time, the app should check if among these people (the voices) there is
one that is known(should have access to a part of my app.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/Microsoft/Cognitive-SpeakerRecognition-Windows/issues/5#issuecomment-287674216>,
or mute the thread
